src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,8 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String size;
+
```

Comment:
if you've added a filed, you must add a constructor, cause without it, the field becomes unreachable